### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Scorp197**
Made changes to all files.

### CODE REVIEW

File: cst438-hw1/src/main/java/cst438/Hello.java [hunk 1]
```diff
@@ -21,8 +21,8 @@
 
    public String createMovieRating(Model model)
    {
-      MovieRating ratings = new MovieRating();
-      model.addAttribute("rating", ratings);
+      MovieRating rating = new MovieRating();
```
Comment (brian-lo0-127001) [inline cst438-hw1/src/main/java/cst438/Hello.java:24]: LGTM

File: cst438-hw1/src/main/resources/templates/rating_show.html [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
 </head>
 <body>
-	<h2 th:inline="text">Hello [[${person.firstName}]] </h2>
+	<h2 th:inline="text">Hello [[${rating.firstName}]] </h2>
 
-	First name: <span th:inline="text"> [[${person.firstName}]] </span> <br />
-	Last name: <span th:inline="text"> [[${person.lastName}]] </span> <br /> 
-	Movie Title: <span th:inline="text"> [[${person.movieTitle}]] </span> <br />
-	Movie Rating: <span th:inline="text"> [[${person.rating}]] </span> <br />
+	First name: <span th:inline="text"> [[${rating.firstName}]] </span> <br />
```
Comment (brian-lo0-127001) [inline cst438-hw1/src/main/resources/templates/rating_show.html:9]: LGTM

File: cst438-hw1/src/main/java/cst438/MovieRating.java [hunk 3]
```diff
@@ -94,4 +96,11 @@ public void setMrating(int mrating)
       this.mrating = mrating;
    };
 
+   public String getDate() {
```
Comment (brian-lo0-127001) [inline cst438-hw1/src/main/java/cst438/MovieRating.java:99]: This is a good idea to add the date function.  LGTM

File: cst438-hw1/src/main/resources/templates/rating_show.html [hunk 4]
```diff
@@ -4,12 +4,12 @@
...
+	Last name: <span th:inline="text"> [[${rating.lastName}]] </span> <br />
```
Comment (brian-lo0-127001) [inline cst438-hw1/src/main/resources/templates/rating_show.html:10]: LGTM

File: cst438-hw1/src/main/resources/templates/rating_show.html (hunk 2 above)
Comment (Scorp197) [reply cst438-hw1/src/main/resources/templates/rating_show.html:9]: Affirmative.

File: cst438-hw1/src/main/java/cst438/MovieRating.java (hunk 3 above)
Comment (Scorp197) [reply cst438-hw1/src/main/java/cst438/MovieRating.java:99]: Affirmative.

File: cst438-hw1/src/main/java/cst438/Hello.java [hunk 5]
```diff
@@ -21,8 +21,8 @@
 
    public String createMovieRating(Model model)
    {
-      MovieRating ratings = new MovieRating();
-      model.addAttribute("rating", ratings);
+      MovieRating rating = new MovieRating();
+      model.addAttribute("rating", rating);
```
Comment (brian-lo0-127001) [inline cst438-hw1/src/main/java/cst438/Hello.java:25]: I think these changes are needed.  Do you think this is ready to be merged?

File: cst438-hw1/src/main/resources/templates/rating_show.html (hunk 4 above)
Comment (Scorp197) [reply cst438-hw1/src/main/resources/templates/rating_show.html:10]: Affirmative.

**Review by brian-lo0-127001 — APPROVED** [review]
Approve

File: cst438-hw1/src/main/java/cst438/Hello.java (hunk 5 above)
Comment (Scorp197) [reply cst438-hw1/src/main/java/cst438/Hello.java:25]: Yes.

**Pull request merged**